fetch.bs

```diff
@@ -1001,11 +1001,34 @@ unset.
 "<code>navigate</code>", or "<code>websocket</code>". Unless stated otherwise, it is
 "<code>no-cors</code>".
 
-<p class="note no-backref">Even though the default <a for=/>request</a>
-<a for=request>mode</a> is "<code>no-cors</code>", standards are highly
-discouraged from using it for new features. It is rather unsafe. "<code>navigate</code>" and
-"<code>websocket</code>" are special values for the HTML Standard.
-[[!HTML]]
+<div class="note no-backref">
+ <dl>
+  <dt>"<code>same-origin</code>"
+  <dd>Used to ensure requests are made to same-origin URLs. <a for=/>Fetch</a> will throw an error
+  if the request is not made to a same-origin URL.
+
+  <dt>"<code>cors</code>"
+  <dd>Ensures that the request being made is a <a>CORS request</a>. Fetch will throw an error if
+  the requested resource does not understand the <a>CORS protocol</a>.
+
+  <dt>"<code>no-cors</code>"
+  <dd>Restricts requests to using <a>CORS-safelisted methods</a> and
+  <a>CORS-safelisted request-headers</a>. Upon success, fetch will return a
+  <a>opaque filtered response</a>.
+
+  <dt>"<code>navigate</code>"
+  <dd>This is a special mode used only when navigating between documents.
+
+  <dt>"<code>websocket</code>"
+  <dd>This is a special mode used only when establishing a WebSocket connection.
+ </dl>
+
+ <p>Even though the default <a for=/>request</a>
+ <a for=request>mode</a> is "<code>no-cors</code>", standards are highly
+ discouraged from using it for new features. It is rather unsafe. "<code>navigate</code>" and
+ "<code>websocket</code>" are special values for the HTML Standard.
```

Comment:
Given that we now mention navigate and websocket above, I don't think we need to mention them here again.